specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations-authoring.json

```diff
@@ -0,0 +1,2703 @@
+        "trainedModels",
+        "nextLink"
+      ],
+      "type": "object",
+      "properties": {
+        "trainedModels": {
+          "description": "Gets or sets the jobs.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TrainedModelObject"
+          }
+        },
+        "nextLink": {
+          "description": "Gets or sets the next page link.",
+          "type": "string"
+        }
+      }
+    },
+    "TrainedModelObject": {
+      "required": [
+        "label",
+        "modelId",
+        "lastTrainedDateTime"
+      ],
+      "type": "object",
+      "properties": {
+        "label": {
+          "description": "Represents trained model label.",
+          "type": "string"
+        },
+        "modelId": {
+          "description": "Represents the Id of this model.\n            The model Id has the format \"{version}-{yyyyMMddThhmmss}-{Guid.NewGuid()}\".\n            This is used almost as a thumbprint for the model and isn't consumed in any of the APIs.",
+          "type": "string"
+        },
+        "lastTrainedDateTime": {
+          "format": "date-time",
+          "description": "Represents trained model last trained time.",
+          "type": "string"
+        }
+      }
+    },
+    "EvaluationResultObject": {
+      "required": [
+        "entityEvaluation",
+        "intentEvaluation"
+      ],
+      "type": "object",
+      "properties": {
+        "entityEvaluation": {
+          "description": "Contains the data related to entity evaluation.",
+          "$ref": "#/definitions/EvaluationResultEntityObject"
+        },
+        "intentEvaluation": {
+          "description": "Contains the data related to intent evaluation.",
+          "$ref": "#/definitions/EvaluationResultIntentObject"
+        }
+      }
+    },
+    "EvaluationResultEntityObject": {
+      "required": [
+        "confusionMatrix",
+        "entities",
+        "microF1",
+        "microPrecision",
+        "microRecall",
+        "macroF1",
+        "macroPrecision",
+        "macroRecall"
+      ],
+      "type": "object",
+      "properties": {
+        "confusionMatrix": {
+          "description": "Represents the confusion matrix between two entities (the two entities can be the same entity)\n            the matrix is between the actual entity that was labelled and the entity that was predicted.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "object",
+            "additionalProperties": {
+              "$ref": "#/definitions/ConfusionMatrixObject"
+            }
+          }
+        },
+        "entities": {
+          "description": "Represents the entities evaluation",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/EvaluationResultIntentAndEntityObject"
+          }
+        },
+        "microF1": {
+          "format": "float",
+          "description": "Represents the micro F1",
```

Comment:
What range should customer expect? Normally this is between 0 and 1 for floats that are effectively a percentage. Please be sure to update the documentation for all floats of this nature so the request and/or response input range is understood.

Reply:
...be sure to specify it in the `description`, though. Using `min` or `max` doesn't hurt, but the Azure SDKs don't use these for documentation nor validate them to avoid tight coupling.